clients/cobol-lsp-vscode-extension/src/__tests__/commands/ClearCopybookCacheCommand.spec.ts

```diff
@@ -14,10 +14,33 @@
 
 import * as path from "path";
 import { clearCache } from "../../commands/ClearCopybookCacheCommand";
+import { Utils } from "../../services/util/Utils";
```

Comment:
This whole test seems completely useless, it takes fake inputs, fake implementations and checks that the function does not fail too much i.e. does not throw.